internal/payloads/reservation_payload.go

```diff
@@ -75,8 +79,8 @@ type AWSReservationResponsePayload struct {
 	// Immediately power off the system after initialization
 	PowerOff bool `json:"poweroff"`
 
-	// Instance IDs, only present for finished reservations
-	Instances []string `json:"instances,omitempty"`
+	// Instances array, only present for finished reservations
+	Instances []InstanceResponse `json:"instances,omitempty"`
 }
```

Comment:
FYI this is what is causing IQE test break, this is a reasonable change so my proposal is to fix OpenAPI and wait for @akhil-jha to prepare IQE PR so we can merge this at once.

Reply:
Sure, tag me in the updated PR